src/Compilers/CSharp/Portable/Symbols/MethodSymbol.cs

```diff
@@ -297,6 +297,13 @@ internal virtual bool IsExplicitInterfaceImplementation
             get { return ExplicitInterfaceImplementations.Any(); }
         }
 
+        // PROTOTYPE: this should become public API
+        /// <summary>
+        /// Indicates whether the method is readonly, i.e.
```

Comment:
Need to clarify this comment. Is it describing whether 

a. The method is declared `readonly` 
b. The method is not declared `readonly` but is a member of a `readonly struct` 

The comment as written is ambigious because it describes the behavior of `this` which could refer to either.